cluster/common.sh

```diff
@@ -995,7 +995,12 @@ function generate-certs {
     mv "kubelet.pem" "pki/issued/kubelet.crt"
     rm -f "kubelet.csr"
 
-    ./easyrsa build-client-full kubecfg nopass
+    # Make a superuser client cert with subject "O=system:masters, CN=kubecfg"
+    ./easyrsa --dn-mode=org \
+      --req-cn=kubecfg --req-org=system:masters \
+      --req-c= --req-st= --req-city= --req-email= --req-ou= \
+      build-client-full kubecfg nopass
```

Comment:
Same as what GCE used to do, except w/ "client" as the cn. I'd rather not mint new client certificates on GKE w/ the system:masters group. The admin client cert is mostly a legacy thing on GKE. And if users want to continue having their "client" user have some permissions, they can add the RBAC binding that allows that (before turning off ABAC).

I don't know what the upgrade story is going to look like on GCE.